src/javascripts/components/largeText.scss

```diff
@@ -0,0 +1,18 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+#checkbox {
+  display: flex;
+  position: fixed;
+  bottom: 1rem;
+  left: 15%;
```

Comment:
Right now, the whole text box is expanding instead of just the font, so it's stretching over the incoming messages area.

If you add width and height in here by pixels, it should stay the same:
```css
width: 640px;
height: 240px;
```

And then in the `addMessage.js` file, you might have to remove the `rows="10" cols="80"` part of the `<textarea>` tag in line 10. That should make it defined by pixels and stay the same size.